lambda/lambda.js

```diff
@@ -100,6 +100,12 @@ async function handler(request) {
                     return;
                 }
                 console.log(jwtKey.modeName + ": payload " + JSON.stringify(payload, null, '  '));
+                // JBA id tokens are valid for an hour and JBT ones for 10 minutes, so the common threshold is set at 3600 seconds 
+                if (Math.floor(Date.now() / 1000) + 3600 < payload.exp) {
```

Comment:
If I do the match right - 1 hour = 3600 seconds. We a change (thanks to time drift) to see a token that is exactly 1 hour to live, so the test would not pass. Please tell me if you could set a longer time-bound here, just in case, say 2 hours?

Reply:
Is this code is safe for time zones? I'm not sure about `Date.now()` and `payload.exp` in that respect.

Reply:
I guess 2 hours is a large enough boundary, especially when Space's tokens are valid for 10 minutes only :)
Indeed, it is better to validate if the difference between `iat` and `exp` claims is greater then 7200 seconds. We will surely be time zone independent then.